src/test/java/seedu/address/testutil/TypicalStudents.java

```diff
@@ -25,44 +25,43 @@
 public class TypicalStudents {
 
     public static final Student ALICE = new StudentBuilder().withName("Alice Pauline")
-            .withId("1")
+            .withId(VALID_ID)
```

Comment:
Does this mean all the students in TypicalStudents have the same ID?

Reply:
I think we can vary the values that VALID_ID has so that the test can be more comprehensive? If not all have the same value of `1`, then the test only cover id of `1`.

Reply:
I try to make Id test very standalone because it's mutable (keeps track of used Id values and auto increment for new students), that's why the TypicalStudents used in other tests will not be affected by Id's mutability and can safely just use a placeholder value (i.e. 1)